### PR TITLE
I1908 rewrite standard edit modal 3

### PR DESCRIPTION
**Pull request by Serhiy91**
Added ReviewsSubcard (client)

### CODE REVIEW

File: app/imports/client/react/reviews/components/ReviewAddFormWrapper.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { EntityManagerForm } from '../../components';
+import ReviewAddContainer from '../containers/ReviewAddContainer';
+
+const LessonAddFormWrapper = props => (
```
Comment (Dartv) [inline app/imports/client/react/reviews/components/ReviewAddFormWrapper.js]: wrong name

**Comment by Dartv** [comment]
You should wrap all components with `memo`

File: app/imports/client/react/reviews/containers/ReviewAddContainer.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+        return createReview({
+          variables: {
+            input: {
+              scheduledDate,
+              reviewedAt,
+              comments,
+              reviewedBy,
+              organizationId,
+              linkedTo: {
```
Comment (Dartv) [inline app/imports/client/react/reviews/containers/ReviewAddContainer.js]: database representation doesn't have `linkedTo` field. Why create confusion?

File: app/imports/client/react/reviews/containers/ReviewEditContainer.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  isOpen: PropTypes.bool.isRequired,
+  toggle: PropTypes.func.isRequired,
+  refetchQueries: PropTypes.func.isRequired,
+  onDelete: PropTypes.func,
+  reviewId: PropTypes.string,
+  review: PropTypes.object,
+  fetchPolicy: PropTypes.string,
+};
+
+export default React.memo(ReviewEditContainer);
```
Comment (Dartv) [inline app/imports/client/react/reviews/containers/ReviewEditContainer.js]: it's better to import `{ memo } from 'react'`

**Comment by Serhiy91** [comment]
@Dartv fixed here https://github.com/Pliohub/Plio/pull/1985

**Pull request merged**